src/org/labkey/test/components/ui/EntityBulkUpdateDialog.java

```diff
@@ -0,0 +1,205 @@
+package org.labkey.test.components.ui;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.glassLibrary.components.FilteringReactSelect;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.ToggleButton;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+
+public class EntityBulkUpdateDialog extends ModalDialog
```

Comment:
Indicate the specific product component that is instrumented by this class.
Also, this seems very similar to the `EntityBulkInsertDialog`. I suspect they are the same product component, with a flag modifying their behavior slightly. If so, this class should extend `EntityBulkInsertDialog` (or vice-versa if that makes more sense).

Reply:
I thought the dialog having enable/disable sliders for editing fields made it different enough from the outside to handle them separately.  If it's necessary to derive one from the other I expect a lot of overrides or forking (to handle the enable/disable field edit sliders) in a lot of places; not sure that will be helpful.

As I look, it seems there are several versions of BulkUpdateForm.d.ts out there, in multiple modules, as well as BulkAddUpdateForm.d.ts as well.  They appear to both use https://github.com/LabKey/labkey-ui-components/blob/master/packages/components/src/components/forms/QueryInfoForm.tsx in common to generate the modal dialog, but use different forms.